platform-configs/pcengines-apu4.robot

```diff
@@ -3,4 +3,5 @@ Resource    include/pcengines.robot
 
 
 *** Variables ***
-${DMIDECODE_PRODUCT_NAME}=      apu4
+${DMIDECODE_PRODUCT_NAME}=          apu4
+${INTERNAL_PROGRAMMER_CHIPNAME}=    W25Q64BV/W25Q64CV/W25Q64FV
```

Comment:
Is this required only for apu4 or for all apus? IIRC flash is the same on all platforms.

In any case, this should be in separate commit, it has nothing to do with the keyword added in the other file.